datagram-socket/src/socket_stats.rs

```diff
@@ -44,6 +44,8 @@ pub trait AsSocketStats {
 pub struct SocketStats {
     pub pmtu: u16,
     pub rtt_us: i64,
+    pub min_rtt_us: i64,
```

Comment:
Do we need to update the package version in order to provide access to these new fields?

Reply:
I think we generally create a separate release commit like this: https://github.com/cloudflare/quiche/commit/603b77b129816fad8211f189f694aa831b1673fc